ctapipe/reco/energy_regressor.py

```diff
@@ -0,0 +1,383 @@
+import numpy as np
+
+from astropy import units as u
+
+from sklearn.ensemble import RandomForestRegressor
+
+
+def pick_any(my_dict):
```

Comment:
• As the docstring says, I don't care which element I get; it doesn't have to be the first, it doesn't have to be random. I'm only interested in interfacing my class to the class of the elements -- and they should all have the same class.

• As far as we understand, `d.values()` creates a new object with a copy of all elements. The regressors can become very big and copying them *all* for this is overkill. If you assure me that `d.values()` does not allocate any unnecessary memory, I can put the `next(iter(…` version inline indeed.